js/style/style_layer.js

```diff
@@ -169,6 +169,20 @@ StyleLayer.prototype = util.inherit(Evented, {
         }
     },
 
+    getPaintValueStopZoomLevels: function(name) {
+        var transition = this._paintTransitions[name];
+        if (transition) {
+            return transition.declaration.getStopZoomLevels();
+        } else {
+            return [];
+        }
+    },
+
+    getPaintInterpolationValue: function(name, zoom) {
```

Comment:
2.333 would mean "interpolate 1/3 of the way from stop 2 to stop 3". It is passed to https://github.com/mapbox/mapbox-gl-js/pull/2454/files#diff-a8436c131e601f997ad9bb209e260f18R12
Names are hard. Any clearer ideas?